lib/libzrepl/data_conn.c

```diff
@@ -1112,6 +1117,29 @@ uzfs_zvol_rebuild_scanner_callback(off_t offset, size_t len,
 	return (0);
 }
 
+void
+uzfs_zvol_send_zio_cmd(zvol_info_t *zinfo, zvol_io_hdr_t *hdrp,
+    zvol_op_code_t opcode, int fd, char *payload, uint64_t payload_size,
+    uint64_t checkpointed_io_seq)
+{
+
+	zvol_io_cmd_t	*zio_cmd;
+	bzero(hdrp, sizeof (*hdrp));
+	hdrp->status = ZVOL_OP_STATUS_OK;
+	hdrp->version = REPLICA_VERSION;
+	hdrp->opcode = opcode;
+	hdrp->checkpointed_io_seq = checkpointed_io_seq;
+	hdrp->len = payload_size; // MAX_NAME_LEN + 1;
+	zio_cmd = zio_cmd_alloc(hdrp, fd);
+	if (payload_size != 0)
+		bcopy(zio_cmd->buf, payload, payload_size);
```

Comment:
is this supposed to be bcopy(src,dst,..) rather than bcopy(dst,src,..)

Reply:
done